src/rfkill/rfkill.c

```diff
@@ -213,11 +228,7 @@ static int load_state(
         if (shall_restore_state() == 0)
                 return 0;
 
-        r = find_device(udev, event, &device);
-        if (r < 0)
-                return r;
-
-        r = determine_state_file(udev, event, device, &state_file);
+        r = determine_state_file(udev, event, &state_file);
```

Comment:
maybe split out this little rearrangement in a separete commit in this PR? (doesn't matter though, it's fine this way, too)